### ISSUE TITLE
GUI is not responsive while protocol is running (Trac #124)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#128]
Pausing a protocol does not work consistently.

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/124
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "Pausing a protocol does not work consistently.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "GUI is not responsive while protocol is running", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-09-27T20:32:32", 
    "milestone": "Microdrop 1.0", 
    "owner": "ryan", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120927T20:38:24: 
* ryan@fobel.net commented:

> commit 0eb73eca2513fe10e773b724e0d28892b4e6fbd0
> Author: Ryan Fobel <ryan@fobel.net>
> Date:   Thu Sep 27 16:32:51 2012 -0400
> 
>     Fixes #128
>     
>     Clicking on any of the protocol buttons didn't always trigger events.
>     Specifically, these events were ignored if the mouse pointer was not
>     over the button widget at the time that the event was processed. This
>     commit uses the location of the mouse pointer at the time that the event
>     was fired.


* ryan@fobel.net changed resolution from "" to "fixed"
* ryan@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "" to "microdrop"
